### ISSUE TITLE
Load failure on initial installation, without configuration

### ISSUE THREAD

**Issue by SlimeDog** [SlimeDog/NetworkInterceptor#18]
**Description**
On initial installation (ie., no configuration files exist), plugin initializes without an installed detection method. Consequently, it fails to load.

**Reproduction steps**
- Install NetworkInterceptor 3.1.3 for first time, so no configuration files exist
- Start the server
- Review console/log

**Expected behavior**
Clean start-up

**Screenshots**
If applicable, add screenshots to help explain the problem

**Server Environment**
 - OS [Linux, MacOS, Windows] version: any
 - Java version [java -version]: 16.0.2
 - Server version [/version]: any
 - NetworkInterceptor version [/version NetworkInterceptor]: 3.1.3

**Additional context**
```
[08:38:26] [Server thread/INFO]: [NetworkInterceptor] Blocking is not enabled
[08:38:26] [Server thread/ERROR]: [NetworkInterceptor] Unknown logging mode: null
[08:38:26] [Server thread/ERROR]: Could not load 'plugins/NetworkInterceptor-3.1.3.jar' in folder 'plugins'
org.bukkit.plugin.InvalidPluginException: me.lucko.networkinterceptor.common.CommonNetworkInterceptor$IllegalConfigStateException: Illegal config value for 'logging.mode': null (Available: [all, console, file])
        at org.bukkit.plugin.java.JavaPluginLoader.loadPlugin(JavaPluginLoader.java:157) ~[patched_1.17.1.jar:git-Paper-325]
        at org.bukkit.plugin.SimplePluginManager.loadPlugin(SimplePluginManager.java:416) ~[patched_1.17.1.jar:git-Paper-325]
        at org.bukkit.plugin.SimplePluginManager.loadPlugins(SimplePluginManager.java:324) ~[patched_1.17.1.jar:git-Paper-325]
        at org.bukkit.craftbukkit.v1_17_R1.CraftServer.loadPlugins(CraftServer.java:419) ~[patched_1.17.1.jar:git-Paper-325]
        at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:287) ~[patched_1.17.1.jar:git-Paper-325]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1212) ~[patched_1.17.1.jar:git-Paper-325]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:319) ~[patched_1.17.1.jar:git-Paper-325]
        at java.lang.Thread.run(Thread.java:831) ~[?:?]
Caused by: me.lucko.networkinterceptor.common.CommonNetworkInterceptor$IllegalConfigStateException: Illegal config value for 'logging.mode': null (Available: [all, console, file])
        at me.lucko.networkinterceptor.common.CommonNetworkInterceptor.setupLoggers(CommonNetworkInterceptor.java:195) ~[NetworkInterceptor-3.1.3.jar:?]
        at me.lucko.networkinterceptor.common.CommonNetworkInterceptor.enable(CommonNetworkInterceptor.java:117) ~[NetworkInterceptor-3.1.3.jar:?]
        at me.lucko.networkinterceptor.common.CommonNetworkInterceptor.<init>(CommonNetworkInterceptor.java:62) ~[NetworkInterceptor-3.1.3.jar:?]
        at me.lucko.networkinterceptor.NetworkInterceptor.<init>(NetworkInterceptor.java:26) ~[NetworkInterceptor-3.1.3.jar:?]
        at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?]
        at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:78) ~[?:?]
        at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?]
        at java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499) ~[?:?]
        at java.lang.reflect.ReflectAccess.newInstance(ReflectAccess.java:128) ~[?:?]
        at jdk.internal.reflect.ReflectionFactory.newInstance(ReflectionFactory.java:350) ~[?:?]
        at java.lang.Class.newInstance(Class.java:642) ~[?:?]
        at org.bukkit.plugin.java.PluginClassLoader.<init>(PluginClassLoader.java:83) ~[patched_1.17.1.jar:git-Paper-325]
        at org.bukkit.plugin.java.JavaPluginLoader.loadPlugin(JavaPluginLoader.java:153) ~[patched_1.17.1.jar:git-Paper-325]
        ... 7 more
```

**Comment by SlimeDog** [comment]
The main configuration file is installed following the load failure, so workaround:
- Shutdown the server
- Edit the configuration to suit
- Start the server again

However, sample configuration files are not installed; they should be. Sample configuration files may be found at https://github.com/SlimeDog/NetworkInterceptor/tree/master/src/main/resources.

**Comment by SlimeDog** [comment]
Resolved by NetworkInterceptor 3.1.4
